### DISCUSSION TITLE
[#1005824] Deprecated Blocks in the Block Selector

### DISCUSSION THREAD

**Discussion by xKingDark** [Mojang/minecraft-editor#101] [Bugs]
**List of deprecated blocks that should be removed**:
> info_update
info_update2
unknown
camera
piston_arm_collision
sticky_piston_arm_collision
glowingobsidian
moving_block
invisible_bedrock
client_request_placeholder_block
reserved6

**Helpful Info**:
- Preview build version: `1.19.80.22`

**Comment by conmaster2112** [comment]
Invisible Bedrock is not deprecated.

**Reply by xKingDark** [reply]
Is it being used in the game?

**Reply by conmaster2112** [reply]
Is barrier block being used in the game?

**Reply by xKingDark** [reply]
The barrier is being used by map makers but no one uses the Invisible Bedrock

**Reply by conmaster2112** [reply]
but i agree with other block types

**Reply by xKingDark** [reply]
The Invisible Bedrock block is unobtainable even with commands

**Comment by conmaster2112** [comment]
Its commonly used BlockType in Script API

**Reply by xKingDark** [reply]
But removing it from the Block Selector doesn’t remove it from the Block Types

**Reply by conmaster2112** [reply]
OK, thx for explenation

**Comment by Xd-pro** [comment]
Some of these blocks are useful for decoration, why remove them? None of them cause any unexpected behaviour

**Reply by xKingDark** [reply]
I don’t see any of them being used in maps, none of the marketplace partners use them in their maps because they do nothing

**Reply by Xd-pro** [reply]
End gateway portals and invisible bedrock are definitely used

**Reply by GameParrot** [reply]
Maybe keep glowing obsidian because it can be used for decoration

**Reply by xKingDark** [reply]
> End gateway portals and invisible bedrock are definitely used

You can use barrier blocks instead of the invisible bedrock

**Reply by Xd-pro** [reply]
You can't target blocks through barriers like you can with invisible bedrock. If you put invisible bedrock in front of a crafting table, you can still use it.

**Reply by conmaster2112** [reply]
> You can't target blocks through barriers like you can with invisible bedrock. If you put invisible bedrock in front of a crafting table, you can still use it.

Ya, Invisible Bedrock has its own functionality💪

**Comment by rpoerschke** [comment]
We addressed this issue in 0.3.3.  We are going to close this discussion, but feel free to re-open if you find additional issues.  Thanks for all your help, @DarkGamerYT
